xchange-hitbtc/src/main/java/org/knowm/xchange/hitbtc/service/HitbtcTradeServiceRaw.java

```diff
@@ -149,12 +149,17 @@ private HitbtcExecutionReportResponse fillHitbtcExecutionReportResponse(LimitOrd
 
   public HitbtcExecutionReportResponse cancelOrderRaw(String orderId) throws IOException {
 
+    return cancelOrderRaw(orderId, "");
```

Comment:
To fix this I took the overloaded method approach. cancelOrderId is an optional field.